docs/contributor/05-20-e2e_tests.md

```diff
@@ -2,81 +2,82 @@
 
```

Comment:
The E2E abbreviation is mentioned in the title. Don't you want to use the full name there as in the previous document?